unikernels/unikraft-chromium-headless/deploy.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+# Function to check if mkfs.erofs is available
+check_mkfs_erofs() {
+    if command -v mkfs.erofs &>/dev/null; then
+        echo "mkfs.erofs is already installed."
+        return 0
+    else
+        echo "mkfs.erofs is not installed."
+        return 1
+    fi
+}
+
+# Function to install erofs-utils package
+install_erofs_utils() {
+    if command -v apt-get &>/dev/null; then
+        echo "Detected Ubuntu/Debian-based system. Installing erofs-utils..."
+        sudo apt update
+        sudo apt install -y erofs-utils
+    elif command -v dnf &>/dev/null; then
+        echo "Detected Fedora-based system. Installing erofs-utils..."
+        sudo dnf install -y erofs-utils
+    elif command -v yum &>/dev/null; then
+        echo "Detected CentOS/RHEL-based system. Installing erofs-utils..."
+        sudo yum install -y erofs-utils
+    elif [[ "$OSTYPE" == "darwin"* ]]; then
+        if command -v brew &>/dev/null; then
+            echo "Detected macOS. Installing erofs-utils..."
+            brew install erofs-utils
+        else
+            echo "Homebrew (brew) not found. Please install Homebrew first."
+            exit 1
+        fi
+    else
+        echo "Unsupported operating system or package manager. Please install erofs-utils manually."
+        exit 1
+    fi
+}
+
+# Stop execution on errors
+set -e
```

Comment:
Locally, I had to manually install the erofs-utils as the script didn't automatically install it. So suggesting a couple changes that _could_ help, but I am not sure. 

I think this line would not be needed, in the context of the other changes I suggested.